### ISSUE TITLE
Peoplepicker suggestions does not have sufficient color contrast in fabric@6.195.1

### ISSUE THREAD

**Issue by luckyyd** [OfficeDev/office-ui-fabric-react#9819]
## Codepen demonstrating the issue
People picker persona initial color in office-ui-fabric-react@6.195.1
https://codepen.io/anon/pen/KjOKma
People picker persona initial color in office-ui-fabric-react@7
https://codepen.io/anon/pen/ewqYEe

## Actual behavior:
The initial persona color in v6 (6.195.1) lts does not meet A11y color contrast requirement: Elements must have sufficient color contrast: Expected contrast ratio of 4.5:1.
 
![image](https://user-images.githubusercontent.com/12855573/61273085-48a59b80-a7db-11e9-9b36-0df82b83451e.png)

![image](https://user-images.githubusercontent.com/12855573/61273148-6f63d200-a7db-11e9-9fd2-9cab1e6b4c5d.png)

However, people picker in fabric v7 adopt a different set of initial colors which meet the color contract ratio requirement.
## Expected behavior:
Update fabric v6 lts so that the people picker persona initial color meet the a11y color contract requirement.

## Documentation describing expected behavior
https://dequeuniversity.com/rules/axe/3.2/color-contrast?application=msftAI
https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html

**Comment by joschect** [comment]
@luckyyd Thanks for submitting this bug! There's currently been some discussion about the right way to go about aligning the palettes that are involved in this. See #9691 and #9692 . This is definitely a bug and we'll keep working on it.

**Comment by micahgodbolt** [comment]
@joschect i see that we decided to move forward with the new colors in Fabric 7. Since we're having contrast issues with the Fabric 6 colors, would it make sense to port those color changes back to 6 as well?

**Comment by micahgodbolt** [comment]
2.211.0 on the left and 6.195.1 on the right

![image](https://user-images.githubusercontent.com/1434956/71699902-a5d7de00-2d76-11ea-8171-0e4738dd0642.png)
 
It looks like we've updated 6.x with the new persona colors.
